deploy/Chart/templates/database/configmaps.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: database-secret
+  namespace: "{{ .Release.Namespace }}"
+  labels:
+    control-plane: database
```

Comment:
Should this be more specific?

Reply:
More specific in what way? This is similar to how we handle labels for other components.